### ISSUE TITLE
Don't build services not to be started

### ISSUE THREAD

**Issue by copitz** [avast/gradle-docker-compose-plugin#104]
Thank you for this great plugin :+1: 

It would be great if services which won't be started also wouldn't be built - f.i. with the following configuration:

```groovy
dockerCompose.isRequiredBy(bootRun)
dockerCompose {
    startedServices = ["virtuoso"]
}
```

and this docker-compose.yml

```yaml
version: '2'
services:
  web:
    image: foo/bar
    build: .
    links:
      - "virtuoso"

  virtuoso:
    image: tenforce/virtuoso
    ports:
      - "1111:1111"
```

the `web` service is currently built on bootRun although its not linked and not started.

**Comment by augi** [comment]
Thanks for the report, the pull request is under review now.

**Comment by augi** [comment]
The change is released in `0.5.9`.

**Comment by copitz** [comment]
Awesome! Thank you!
